webservice/model/form.go

```diff
@@ -42,22 +43,38 @@ type FormRepository struct {
 // Insert form to database
 func (repo *FormRepository) Insert(form Form) (int, error) {
 
+	tx, err := db.GetDB().Begin() //start transaction
+	if err != nil {
+		log.Println(err.Error())
+		return -1, err
+	}
+
 	// Get current Norwegian time in string format TODO time-norwegian
 	date := util.ConvertTimeStampToString(util.GetTimeInCorrectTimeZone())
 
 	// Insertions Query
 	query := "INSERT INTO forms (prefix, name, description, created) VALUES (?, ?, ?, ?);"
 	// Execute query with parameters
-	rows, err := db.GetDB().Exec(query, form.Prefix, form.Name, form.Description, date)
+	rows, err := tx.Exec(query, form.Prefix, form.Name, form.Description, date)
 	// Check for error
 	if err != nil {
+		log.Fatal(err.Error())
+		tx.Rollback() //quit transaction if error
 		return -1, err
 	}
 
 	// Get last inserted id from table
 	id, err := rows.LastInsertId()
 	// Check for error
 	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
No need for logging inside here, cuase error is a return value.
And don't use log.Fatal() cauase it will stop the whole application.